main/level_1.gd

```diff
@@ -2,15 +2,39 @@ extends Node2D
 
 @onready var tilemap: TileMap = get_node("TileMap")
 @onready var camera: Camera2D = get_node("Player/Camera2D")
+@onready var enemy_scene: PackedScene = preload("res://enemies/Enemy.tscn")
+@onready var tile_set: TileSet
  
 func _ready() -> void:
+  tile_set = tilemap.tile_set
```

Comment:
How is this working, the code it's not formatted :)

Reply:
😅 fixed